### ISSUE TITLE
sc5-styleguide v. 0.3.40 doesnt work

### ISSUE THREAD

**Issue by rasmusdamberg** [dwarfhq/frontend-scaffold#7]
There is an issue generating the styleguide. It seems the v. 0.3.40 of sc5-styleguide is broken in some way, related to our scaffold.

**Comment by twixier** [comment]
I have tested both 0.3.40 and 0.3.41 - still same result. I have an idea that sc5 is missing some file of a kind - else i suspect it might have something to do with node. Anyhow - I need some more time to do a thorough debugging session. In the mean time I think im gonna make a rollback for sc5 to 0.3.37 that we know works. 

@rasmusdamberg Do you agree on that solution for now?

**Comment by twixier** [comment]
@rasmusdamberg I have found the error that prevented sc5 to work as intended. I have added an overviewPath to the config.json file to allow for a "frontpage" or "description" page for the styleguide. Its now defaulting to "Readme.md" in the root folder. 

Will you take a run at it and see if that fixes your problems? If so, i will make a full update of the required modules later on to provide the newest updates for the scaffold
